recipes-openxt/openxt/openxt-measuredlaunch_1.0.bb

```diff
@@ -6,19 +6,22 @@ inherit xenclient
 
 SRC_URI = " \
     file://ml-functions \
-    file://seal-system \
+    file://recovery-method \
+
```

Comment:
The code(script) was tested before submitting. The recipe change was minor and inspection is sufficient.